src/libmongoc/tests/unified/runner.c

```diff
@@ -596,7 +596,7 @@ get_topology_type (mongoc_client_t *client)
 static void
 check_schema_version (test_file_t *test_file)
 {
-   const char *supported_version_strs[] = {"1.8"};
+   const char *supported_version_strs[] = {"1.8", "1.12"};
```

Comment:
```suggestion
   const char *supported_version_strs[] = {"1.8", "1.12" /* 1.12 is partially supported to run tests with `errorResponse` */};
```

The Unified Test Format [Changelog](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#changelog) suggests there are operations in older versions not yet supported (e.g. the `thread` entities added in 1.10).